src/main/java/org/generationcp/middleware/pojos/workbench/CropPersonId.java

```diff
@@ -0,0 +1,53 @@
+package org.generationcp.middleware.pojos.workbench;
+
+import org.apache.commons.lang3.builder.EqualsBuilder;
+import org.apache.commons.lang3.builder.HashCodeBuilder;
+import org.generationcp.middleware.pojos.Person;
+
+import java.io.Serializable;
+
+public class CropPersonId implements Serializable {
```

Comment:
Why not CropPersonDTO?

Reply:
This is actually used as a Hibernate composite primary key class for CropPerson.